app/(saas)/discover/_features/trending/trending.tsx

```
@@ -0,0 +1,42 @@
+import { Section } from "@/app/(saas)/discover/_components/section/section";
+
+import { ProjectReactQueryAdapter } from "@/core/application/react-query-adapter/project";
+
+export function Trending() {
+  const { data, isLoading, isError } = ProjectReactQueryAdapter.client.useGetProjectsV2({
+    queryParams: { pageSize: 5 },
+  });
+
+  if (isError) return null;
+
+  const renderProjects = useCallback(() => {
```

ESLint

[error] React Hook "useCallback" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)